Tests/fixtures/tablet-11.yml

```
@@ -8253,3 +8253,75 @@
     model: Volta 5
   os_family: Android
   browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 7.0; CONTIXO CX1065 3G) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Mobile Safari/537.36
+  os:
+    name: Android
+    version: "7.0"
+    platform: ""
+  client:
+    type: browser
+    name: Chrome Mobile
+    version: 119.0.0.0
+    engine: Blink
+    engine_version: 119.0.0.0
+  device:
+    type: tablet
+    brand: Contixo
+    model: CX-1065 3G
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; arm_64; Android 12; TGR-W09) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.6367.508 YaSearchBrowser/24.65/apad BroPP/1.0 YaSearchApp/24.65/apad webOmni SA/3 Mobile Safari/537.36
+  os:
+    name: Android
+    version: "12"
+    platform: ARM
+  client:
+    type: browser
+    name: Yandex Browser
+    version: "24.65"
+    engine: Blink
+    engine_version: 124.0.6367.508
+  device:
+    type: tablet
+    brand: Huawei
+    model: MatePad 11.5" S WiFi
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Linux; Android 11; TPC-G1011LTE) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36
+  os:
+    name: Android
+    version: "11"
+    platform: ""
+  client:
+    type: browser
+    name: Chrome
+    version: 127.0.0.0
+    engine: Blink
+    engine_version: 127.0.0.0
+  device:
+    type: tablet
+    brand: JAY-Tech
+    model: Tablet PC G1011LTE
```

```suggestion
    model: Tablet PC G 10.11" LTE
```

or `G10.11 LTE`

`https://www.norma24.de/android-tablet-pc-g10.11lte-16gb`